alert.tf

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Create an alert policy based on the uptime check.
+resource "google_monitoring_alert_policy" "uptime_alert" {
```

Comment:
I think we need a way of plumbing notification channels into here for this to be useful.

Reply:
Yeah that'll happen in the next PR.

Reply:
Done, that was easy.